package.json

```diff
@@ -62,6 +62,7 @@
     "docs-lint": "npm-run-all --aggregate-output --parallel docs-vnu docs-linkinator",
     "docs-serve": "hugo server --port 9001 --disableFastRender",
     "docs-serve-only": "npx sirv-cli _site --port 9001",
+    "docs-spellcheck": "cspell \"site/**/*.md\"",
```

Comment:
This isn't included in the `docs` script thus it doesn't run on CI. Also, I was thinking if we should use an action but maybe lets leave this for later.

We have plenty of devDependencies and I'm experimenting with moving some of them to actions.

Reply:
Oh, and perhaps move the config file into the build folder?

Reply:
An advantage of keeping the config in the root of the project is that it'll be used automatically for contributors who have the cSpell VS Code extension installed - which I think is helpful.

Yep, I didn't add the 'docs-spellcheck' to 'docs-lint' in the PR as there are a few typo errors thrown as I mentioned above (just some latin dummy text and one nonblocking reference).. but agree that it should be added to `docs-lint`.

Agree that there are a lot of devDependencies and it could be useful to look into using a GitHub Action for this.

Reply:
OK, let's leave the config in root, although I also try not to because the root folder becomes bloated.

Please add the script in `docs-lint` and we can land this after the issues are fixed.

Reply:
ok done!

Reply:
Does cspell have a silent option?

Reply:
yep, just saw this:
https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#options

Reply:
I think we probably need `--no-progress`

Reply:
`"docs-spellcheck": "cspell \"site/**/*.md\" -s",`
just tested locally, seems to work OK - but if it fails it doesn't tell you what the error (typo) was or where. Up to you if you want to add that or not...

Reply:
ah - I think we want:
`"docs-spellcheck": "cspell \"site/**/*.md\" --no-progress",`